plugins/bulk_upload/filter/lib/api/KalturaBulkServiceFilterDataOnlyFilter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Represents the Bulk service input for filter bulk upload
+ * @package plugins.bulkUploadFilter
+ * @subpackage api.objects
+ */
+class KalturaBulkServiceFilterDataOnlyFilter extends KalturaBulkServiceData

```

Comment:
rename to KalturaBulkUploadFilterBase

Reply:
Rename to KalturaBulkServiceFilterDataBase